aws/resource_aws_sqs_queue.go

```diff
@@ -187,7 +188,18 @@ func resourceAwsSqsQueueCreate(d *schema.ResourceData, meta interface{}) error {
 		req.Attributes = attributes
 	}
 
-	output, err := sqsconn.CreateQueue(req)
+	var output *sqs.CreateQueueOutput
+	err := resource.Retry(1*time.Minute, func() *resource.RetryError {
+		var err error
+		output, err = sqsconn.CreateQueue(req)
+		if err != nil {
+			if isAWSErr(err, sqs.ErrCodeQueueDeletedRecently, "You must wait 60 seconds after deleting a queue before you can create another with the same name.") {
```

Comment:
Nitpick, but I'd perhaps loosen the match here a little bit, just in case they raise the limit from 60 secs, i.e.

`after deleting a queue before you can create another with the same name.`

or maybe even

`You must wait `

Reply:
If they change the limit, then we should adjust our retry threshold to match it. Our daily acceptance testing will start failing since this conditional will always return a `NonRetryableError`

Reply:
Ah, very good point 👌   - ignore me then! 😅